diff --git a/node_modules/react-awesome-styled-grid/lib/react-awesome-styled-grid.cjs.js b/node_modules/react-awesome-styled-grid/lib/react-awesome-styled-grid.cjs.js
index 7c75d33..000f81c 100644
--- a/node_modules/react-awesome-styled-grid/lib/react-awesome-styled-grid.cjs.js
+++ b/node_modules/react-awesome-styled-grid/lib/react-awesome-styled-grid.cjs.js
@@ -762,7 +762,7 @@ Col.propTypes = {
 };
 
 function useEventListener (e, r, i) {
-  void 0 === i && (i = global);
+  void 0 === i && (i = window);
   var o = React.useRef();
   React.useEffect(function () {
     o.current = r;
diff --git a/node_modules/react-awesome-styled-grid/lib/react-awesome-styled-grid.es.js b/node_modules/react-awesome-styled-grid/lib/react-awesome-styled-grid.es.js
index 5fa28e2..341bd7f 100644
--- a/node_modules/react-awesome-styled-grid/lib/react-awesome-styled-grid.es.js
+++ b/node_modules/react-awesome-styled-grid/lib/react-awesome-styled-grid.es.js
@@ -754,7 +754,7 @@ Col.propTypes = {
 };
 
 function useEventListener (e, r, i) {
-  void 0 === i && (i = global);
+  void 0 === i && (i = window);
   var o = useRef();
   useEffect(function () {
     o.current = r;
